### PR TITLE
Feature/upp update

### PR DESCRIPTION
**Pull request by Hiroki6**

### CODE REVIEW

File: event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala [hunk 1]
```diff
@@ -111,113 +111,66 @@ class EncoderExecutor @Inject() (
...
-          }
-
-        payload <- Try(EncoderJsonSupport.ToJson[ProtocolMessage](messageEnvelope.ubirchPacket).get)
-
-        eventLog <- Try(messageEnvelope)
-          .map(_.ubirchPacket)
-          .filter(_.getHint == 0)
-          .map { _ =>
-
-            val payloadJsNode = Option(messageEnvelope)
```
Comment (Hiroki6) [inline event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala:134]: this part creating each field were replaced with helper methods in the EncodeHelper object.

Comment (dermicha) [reply event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala:134]: you use the wrong topic ubirch-svalbard-evt-log-json, delete/disable/enable UPPs should be send to a new topic

Comment (lalottacontinua) [reply event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala:134]: I like the EncodeHelper very much. It makes it much more readable. Couldn't the customerId retrieval also get moved to the helper?

Comment (lalottacontinua) [reply event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala:134]: and the getCategory method maybe as well?

Comment (Hiroki6) [reply event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala:134]: > you use the wrong topic ubirch-svalbard-evt-log-json, delete/disable/enable UPPs should be send to a new topic

yes. I'll change it after I add new topic on the helm chart and it's created on dev.

Comment (Hiroki6) [reply event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala:134]: > I like the EncodeHelper very much. It makes it much more readable. Couldn't the customerId retrieval also get moved to the helper?

ah, yes. I'll move it. the getCategory method as well. thanks

**Review by lalottacontinua — APPROVED** [review]

**Pull request merged**